src/faldbt/lib.py

```diff
@@ -1,36 +1,55 @@
 # NOTE: COPIED FROM https://github.com/dbt-labs/dbt-core/blob/40ae6b6bc860a30fa383756b7cdb63709ce829a8/core/dbt/lib.py
-from dbt.adapters.factory import Adapter
-from dbt.contracts.graph.manifest import Manifest
 import os
-from collections import namedtuple
-from faldbt.cp.contracts.graph.parsed import ParsedModelNode
-from faldbt.cp.contracts.sql import ResultTable, RemoteRunResult
+import six
+
 from datetime import datetime
 from uuid import uuid4
+from collections import namedtuple
+from typing import List, Type, Union
 
+from dbt.adapters.factory import Adapter
+from dbt.adapters.protocol import ConnectionManagerProtocol
+from dbt.config.runtime import RuntimeConfig
+from dbt.contracts.graph.manifest import Manifest
+import dbt.clients.agate_helper as agate_helper
+import dbt.adapters.factory as adapters_factory
+
+import pandas as pd
+
+from sqlalchemy.sql.ddl import CreateTable
```

Comment:
looks like there are different dialects for each different source 

https://github.com/googleapis/python-bigquery-sqlalchemy

Reply:
Yes, I believe we generate for sqlite for now. Should we "match" the correct dialect in this PR or as a future task?